cuda_core/examples/device_properties.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# SPDX-License-Identifier: LicenseRef-NVIDIA-SOFTWARE-LICENSE
+
+from cuda.core.experimental import Device, system
+
+
+def _bool_str(value: bool) -> str:
```

Comment:
`_yes_or_no` (or just `_yes_no` works, too)

makes this self-explanatory (in the call sites).

Reply:
Yep, will change both filename and function to `_yes_no`